pages/admin/AdminFunctionsHandler.inc.php

```diff
@@ -109,6 +109,31 @@ function clearDataCache() {
 
 		Request::redirect(null, 'admin');
 	}
+
+	/**
+	 * Download scheduled task execution log file.
+	 * @param $args array
+	 */
+	function downloadScheduledTaskLogFile() {
+		$this->validate();
+		$application =& Application::getApplication();
+		$request =& $application->getRequest();
+
+		$file = $request->getUserVar('file');
```

Comment:
This might need sanitizing. True, this is a site administrator function, but you could effectively download anything off the server this way, I think.